corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -68,8 +68,12 @@ def handle(self, existing_domain, new_domain, **options):
         if self._clone_type(options, 'fixtures'):
             self.copy_fixtures()
 
-        if self._clone_type(options, 'locations'):
-            types_only = self._clone_type(options, 'types_only')
+        copy_locations = self._clone_type(options, 'locations')
+        copy_location_types = self._clone_type(options, 'location_types')
+        if copy_locations and not copy_location_types:
+            raise CommandError("You can't copy locations by excluding types")
+        else:
+            types_only = copy_location_types and not copy_locations
             self.copy_locations(types_only)
```

Comment:
it looks like this now calls `self.copy_locations` even when both `copy_locations` and `copy_location_types` are both false. should this check if `copy_locations` is true before copying?